src/components/test/input-disabled.njk

```diff
@@ -0,0 +1,35 @@
+<main>
+
+  <label class="usa-label" for="input-type-text">Text input</label>
+  <input class="usa-input disabled" id="input-type-text" name="input-type-text" type="text">
+
+  <label class="usa-label" for="input-type-text">Text input disabled</label>
+  <input class="usa-input" disabled id="input-type-text" name="input-type-text" type="text">
+
+  <label class="usa-label" for="options">Dropdown</label>
+  <select class="usa-select" name="options" id="options">
+    <option value>- Select -</option>
+    <option value="value1">Option A</option>
+    <option value="value2">Option B</option>
+    <option value="value3">Option C</option>
+  </select>
+
+  <label class="usa-label" for="options">Dropdown Disabled</label>
+  <select class="usa-select" disabled name="options" id="options">
+      <option value>- Select -</option>
+      <option value="value1">Option A</option>
+      <option value="value2">Option B</option>
+      <option value="value3">Option C</option>
+  </select>
+
+  <div class="usa-form-group">
+    <label class="usa-label" for="file-input-single">Input accepts a single file</label>
+    <input id="file-input-single" class="usa-file-input" type="file" name="file-input-single" />
+  </div>
+
+  <div class="usa-form-group">
+      <label class="usa-label" for="file-input-single">Input accepts a single file</label>
+      <input id="file-input-single" class=" usa-file-input usa-file-input--disabled usa-file-input__box" type="file" name="file-input-single" />
+  </div>
+
+</main>
```

Comment:
@mahoneycm was this meant to be included?

Reply:
@mejiaj no my apologies! Was working on something in a different branch and thought I omitted that from my commit

Reply:
All good, thank you!